### ISSUE TITLE
Make it work in Jenkins Scripted Pipeline

### ISSUE THREAD

**Issue by jamesamurr-bind** [jenkinsci/aws-cloudwatch-logs-publisher-plugin#17]
I'd like to use this in the scripted pipeline, but it looks like it only works on the declarative pipeline (scripted pipeline users can't utilize this plugin).

**Comment by njtman** [comment]
I don't believe this can be used in the declarative pipeline either. A pull request which added pipeline support was merged in master on August 16th 2018 but a new version of the plugin was never released.  The latest version of the plugin was released April 2018.

**Comment by jamesamurr-bind** [comment]
Oh! Well I hope that gets released. Appreciate the update.
